docs/dev/connectors/kafka.md

```diff
@@ -100,6 +108,31 @@ Note that the streaming connectors are currently not part of the binary distribu
 * Follow the instructions from [Kafka's quickstart](https://kafka.apache.org/documentation.html#quickstart) to download the code and launch a server (launching a Zookeeper and a Kafka server is required every time before starting the application).
 * If the Kafka and Zookeeper servers are running on a remote machine, then the `advertised.host.name` setting in the `config/server.properties` file must be set to the machine's IP address.
 
+## Modern Kafka Connector
```

Comment:
I am wondering if instead of "Modern" we should just call it "unversioned".

Reply:
I don't object, what do you think? cc @aljoscha

Reply:
Both of those names can be confusing for the user. Maybe we should name this section as `Kafka 0.11+ Connector`?